### ISSUE TITLE
log spam with mutedplayersregex

### ISSUE THREAD

**Issue by nwesterhausen** [nwesterhausen/valheim-discordconnector#111]
somewhat frequently the log is spammed with an error "at DiscordConnector.Config.ManConfig.get_MutedPlayersRegex ()"

![unknown](https://user-images.githubusercontent.com/2317381/174435145-83e3f9cc-770c-4fc5-8c50-0ed4e34dc062.png)

**Comment by nwesterhausen** [comment]
From NexusMods post:

> When someone logs into the server this error happens

```
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in 
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in 
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in <28706b76ffcb4becb5d98adada583fbb>:0 
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in <28706b76ffcb4becb5d98adada583fbb>:0 
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in <28706b76ffcb4becb5d98adada583fbb>:0 
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in <28706b76ffcb4becb5d98adada583fbb>:0 
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in <28706b76ffcb4becb5d98adada583fbb>:0 
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in <28706b76ffcb4becb5d98adada583fbb>:0 
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in <28706b76ffcb4becb5d98adada583fbb>:0 
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in <28706b76ffcb4becb5d98adada583fbb>:0 
  at DiscordConnector.Config.MainConfig.get_MutedPlayersRegex () [0x00000] in <28706b76ffcb4becb5d98adada583fbb>:0 
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Delegate.DynamicInvokeImpl (System.Object[] args) [0x000e7] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) [0x00008] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Delegate.DynamicInvoke (System.Object[] args) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at RoutedMethod`4[T,U,V,B].Invoke (System.Int64 rpc, ZPackage pkg) [0x0001d] in <31c1e64c47f44dec92a720a382fd1940>:0 
  at ZRoutedRpc.HandleRoutedRPC (ZRoutedRpc+RoutedRPCData data) [0x00022] in <31c1e64c47f44dec92a720a382fd1940>:0   at ZRoutedRpc.RPC_RoutedRPC (ZRpc rpc, ZPackage pkg) [0x00023] in <31c1e64c47f44dec92a720a382fd1940>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <eae584ce26bc40229c1b1aa476bfa589>:0 
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Delegate.DynamicInvokeImpl (System.Object[] args) [0x000e7] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) [0x00008] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Delegate.DynamicInvoke (System.Object[] args) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at ZRpc+RpcMethod`1[T].Invoke (ZRpc rpc, ZPackage pkg) [0x0001d] in <31c1e64c47f44dec92a720a382fd1940>:0 
  at (wrapper dynamic-method) ZRpc.DMD<ZRpc::HandlePackage>(ZRpc,ZPackage)
  at ZRpc.Update (System.Single dt) [0x0003e] in <31c1e64c47f44dec92a720a382fd1940>:0 
```

**Comment by nwesterhausen** [comment]
Fixed in release v1.6.1
